tests/network_test.cpp

```diff
@@ -76,6 +76,50 @@ TEST_F(NetworkTest, transfer_block) {
   ASSERT_EQ(1, num_received);
 }
 
+// Test creates two Network setup and verifies sending blocks
+// between is successfull
+TEST_F(NetworkTest, transfer_lot_of_blocks) {
+  auto node_cfgs = make_node_cfgs<20, true>(2);
+  auto nodes = launch_nodes(node_cfgs);
+
+  std::vector<std::shared_ptr<DagBlock>> dag_blocks;
+  std::vector<trx_hash_t> trx_hashes;
+  std::vector<taraxa::bytes> trx_bytes;
+
+  // creating a lot of trxs
+  auto trxs = samples::createSignedTrxSamples(0, 1500, g_secret);
+  for (const auto& trx : trxs) {
+    trx_bytes.emplace_back(*trx.rlp());
+    trx_hashes.push_back(trx.getHash());
+  }
+
+  // creating lot of non valid blocks just for size
+  for (int i = 0; i < 100; ++i) {
+    DagBlock blk(blk_hash_t(1111 + i), 0, {blk_hash_t(222 + i), blk_hash_t(333 + i), blk_hash_t(444 + i)}, trx_hashes,
+                 sig_t(7777 + i), blk_hash_t(888 + i), addr_t(999 + i));
+    dag_blocks.emplace_back(std::make_shared<DagBlock>(blk));
+  }
```

Comment:
Why need those 100 invalid DAG blocks? Just for occupy size of sendBlocks. I think the unit test for testing sendBlocks could be able to split large packet. But how do we know if the total size is greater than 16MB or not?

Reply:
yes I need 100 blocks with 1.5k trxs so SendBlocks function has to split it. I know it because when you enable logging you can see msg, that it is splitting the packet